### PR TITLE
Mobile version of My potfolio website

### PR DESCRIPTION
**Pull request by jonas-45**
My potfolio is a mobile version only website with a status bar, a logo, navigation overflow icon and a header.

### CODE REVIEW

**Comment by Yazino12** [comment]
## Hi @jonas-45 🖐🏾,

You have done a great job putting such a highly organized project 🎯, yet I have to invalidate this one for the reasons mentioned below 👇🏽👇🏽👇🏽

### Invalidation Reasons:
- Please make sure to add the section **Future Features** in your README, including 1-3 potential features to be added.  _The only **optional** section is FAQ_
- Make sure to remove the **template logo**. Optionally, you can replace it with your own logo if you have one, _**and make sure it works**_.
- Please consider adding a short summary describing to your pull request body about introduced changes in more detail. **_You can use bullet points_**.

# STATUS: INVALID CODE REVIEW REQUEST 🚫🚫🚫
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jonas-45** [comment]
Changes introduced in this pull request

.Future features to be introduced include expanding it to include a desktop version, adding details of my services and expertise in modals and also adding a contact form to enable me connect to my visitors.

.Readme file has also be updated with the removal of a logo image to be replaced with a personal logo in the future.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <title>Document</title>
+
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div class="container">
+        <header class="app-bar">
+            <div class="status-bar"><img alt="status bar" src="images/statusbar.png"></div>
```
Comment (Abenezer-Tilahun) [inline index.html:16]: - [ Optional ] Good Job for stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). But in my opinion "Status-bar" is not required , So kindly remove it.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <nav>
+                <a href="index.html"><h2>Jonas</h2></a>
+                <div>
+                    <a><img alt="overflow-image" src="images/icon_menu_img.png"></a>
+                </div>
+            </nav>
+        </header>
+
+        <section class="header-section">
+            <h1 class="title">I'm Jonas <br>Glad to see you</h1>
```
Comment (Abenezer-Tilahun) [inline index.html:26]: - [ Optional ] Good Job for stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So kindly adjust the alignment of the title just like the figma design. 

Your Project Look likes this : 

Capture

The Figma design Look likes this : 

Capture

File: index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                            <img src="images/medium.svg" alt="medium handle">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_self">
+                            <img src="images/angellist.svg" alt="angellist handle">
+                        </a>
+                    </li>
+                </ul>
+            </div>
```
Comment (Abenezer-Tilahun) [inline index.html:55]: - [ Optional ] Kindly I recommend you  to add the reaming "GitHub" social icons just like the figma design .

Your Social Icons Look likes this : 

Capture

The Figma Social Icons Look likes this : 

Capture

File: README.md [hunk 4]
```diff
@@ -1,2 +1,102 @@
...
+- [Future Features](#future-features)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
```
Comment (Abenezer-Tilahun) [inline README.md:27]: - [ Optional ] Kindly update the PROJECT DESCRIPTION title with "Portfolio".

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @jonas-45  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (jonas-45) [reply index.html:26]: Thank you. Will correct it right away

**Pull request merged**